src/routes/chat/Chat.jsx

```diff
@@ -30,13 +48,17 @@ const ChatContent = () => {
 
 const Chat = () => {
   const { chatRoomId } = useParams();
+  const [isViewingUnArchived, setViewingUnArchived] = useState(true);
```

Comment:
camelcase properly and follow useState naming convention, rename to `isViewingUnarchived, setIsViewingUnarchived`.